In [1]:
pwd

'C:\\Users\\LENOVO\\18082020'

### Please look in the files mentioned below under 'site-packages'

#### http://localhost:8888/tree/18082020/site-packages


#### * Sec XVII - CSV files

**C:\\Users\\LENOVO\\18082020\\Sec XVII - PDFs and CSV files\\Sec XVII - Working with csv files in Python.ipynb**


#### * Sec XVII - PDFs files

**C:\Users\LENOVO\18082020\Sec XVII - PDFs and CSV files\Sec XVII - Working with pdf files in Python.ipynb**